src/views/AddItem.jsx

```diff
@@ -96,7 +97,7 @@ export function AddItem({ listToken, itemList, setData }) {
 								value="30"
 								onChange={handleTime}
 							/>
-							Not Soon (30 days)
+							Next month
```

Comment:
May be it is better to put more general 'later' as @michellerenehey suggested on Slack?

Reply:
I think 'next month' is fine! I like that it all fits together: this week, this month, next month. Feels clean. 

 I was thinking more about the dilemma and came to a brain conclusion:  If a user puts 'next month' and then the month passes (or, well, 30+ days pass... 45 days pass....60 days pass), then it's no longer 'next month' and it will make sense for it to be inactive!